src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/package.scala

```diff
@@ -14,19 +14,37 @@ package object config {
   }
 
   implicit val dataprocConfigReader: ValueReader[DataprocConfig] = ValueReader.relative { config =>
-    DataprocConfig(config.getString("serviceAccount"),
-      config.getString("dataprocInitScriptURI"),
+    DataprocConfig(config.getString("applicationName"),
```

Comment:
It's a good thing the 22-member-limit for case classes was removed! :)
https://issues.scala-lang.org/browse/SI-7296

Reply:
Yeah - was thinking we could maybe break out dataproc config at some point